languages/messages/wikia/crowdin.conf

```diff
@@ -0,0 +1,4 @@
+[crowdin.project]
+projectid = "CoreMessagesWikia"
+format = "mediawiki"
+source.file = "MessagesEn.php"
```

Comment:
And actually, upload won't work either, since the script looks for the `en` key in the messages array, which doesn't apply to this file.